app/controllers/Controller.scala

```diff
@@ -31,31 +31,25 @@ trait Controller extends PlayController
   implicit def AuthenticatedRequest2Request[T](r: SecuredRequest[T]): Request[T] =
     r.request
 
-  def ensureTeamAdministration(user: User, teamId: ObjectId): Fox[Unit] =
-    for {
-      teamIdBson <- teamId.toBSONObjectId.toFox
-      _ <- ensureTeamAdministration(user, teamIdBson)
-    } yield ()
+  def ensureTeamAdministration(user: UserSQL, teamId: ObjectId): Fox[Unit] =
+    Fox.assertBoolean(user.isTeamManagerOrAdminOf(teamId)) ?~> Messages("team.admin.notAllowed")
```

Comment:
Does `assertBoolean` actually assert it's true? It looks like it should, but it doesn't sound like it does.

Reply:
I sure hope it does (see above)